token/services/ttx/manager.go

```diff
@@ -18,32 +18,32 @@ import (
 	"github.com/pkg/errors"
 )
 
-type TTXDBProvider interface {
+type DBProvider interface {
 	DBByTMSId(id token.TMSID) (*ttxdb.DB, error)
 }
 
 // Manager handles the databases
 type Manager struct {
 	networkProvider NetworkProvider
-	ttxdbProvider   TTXDBProvider
+	ttxdbProvider   DBProvider
 
 	storage storage.DBEntriesStorage
 	mutex   sync.Mutex
-	owners  map[string]*Owner
+	owners  map[string]*DB
 }
 
-// NewManager creates a new Owner manager.
-func NewManager(np NetworkProvider, ttxdbManager TTXDBProvider, storage storage.DBEntriesStorage) *Manager {
+// NewManager creates a new DB manager.
+func NewManager(np NetworkProvider, ttxdbManager DBProvider, storage storage.DBEntriesStorage) *Manager {
 	return &Manager{
 		networkProvider: np,
 		storage:         storage,
 		ttxdbProvider:   ttxdbManager,
-		owners:          map[string]*Owner{},
+		owners:          map[string]*DB{},
```

Comment:
Maybe we can rename this field to txDbs or something similar.

Reply:
👍